bpfman/src/types.rs

```diff
@@ -398,6 +491,16 @@ impl ProgramData {
         )
     }
 
+    /// Retrieves the kind of program.
```

Comment:
as specified by the (ProgramType)[https://github.com/bpfman/bpfman/blob/main/bpfman/src/types.rs#L1534] structure

Reply:
Just to be clear: The preference is for the text to read:

"Retrieves the kind of program as specified by [the] (ProgramType)[https://github.com/bpfman/bpfman/blob/main/bpfman/src/types.rs#L1534]"?

Reply:
Well, not with the line number because that will obviously change over time.

Reply:
Done in 1cd04b01359f2f5a27acdf9947c20c00981f691b.